html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action=>
+    <label for="email" >Email</label>
+    <input type="email" id="email" name="email"/>
+
+
+    <label for="why-coding">Why is coding cool ?</label>
+    <select name="options" id="why-coding">
+    <option value="Interesting"> Because it's interesting</option>
+    <option value="fun">Because it's fun</option>
+    <option value="Powerful">Because it's powerful</option>
+    </select>
+
+
+    <label for="fav-lang" >Favourite language (choose one)</label>
+    <input type="radio" name="language" id= "fav-lang"value="HTML" check="checked"/> HTML
```

Comment:
You should _**never**_ have more than one element with the same `id` in your HTML document. In this case, the `label` shouldn't be the text above the radio inputs, it should be the text following the radio inputs. 

Please place "HTML", "CSS", and "JavaScript" in `label` elements, and revise each radio input to have a unique `id` to match their `label`.